src/borg/repository.py

```diff
@@ -505,10 +505,13 @@ def _load_hints(self):
 
     def info(self):
         """return some infos about the repo (must be opened first)"""
-        self._load_hints()
-        return dict(
-            id=self.id, version=self.version, append_only=self.append_only, storage_quota_use=self.storage_quota_use
-        )
+        info = dict(id=self.id, version=self.version, append_only=self.append_only)
+        try:
+            self._load_hints()
+        except TypeError:
+            pass
```

Comment:
why is that needed?

Reply:
Because when working with a RemoteRepository, _load_hints raises a TypeError

Reply:
yeah, I already noted that it can't work that way. but the solution is not to suppress the error (aka "diaper pattern", catches all the shit :-) ), but to use the .info() method instead of an attribute access.

Reply:
(to correct myself: `try: ... except: ...` is the diaper pattern, but the idea is the same)

Reply:
I can't use .info() method there because that is info().

Reply:
the code in repository.py always uses a local repository (relative from where that code is executed).

in case of a remote repo, the code in remote.py runs on the client and the code in repository.py runs on the repo server.

so, please remove the exception handler and analyse the root cause of whatever issue you hid by it.